### PR TITLE
Edit and extend the text

### PR DESCRIPTION
**Pull request by petrochenkov**

### CODE REVIEW

File: default auto traits.md [hunk 1]
```diff
@@ -61,27 +71,29 @@ or
 trait Trait {
    type Type where Self: Sized;
 }
-trait Trait2<T> : Trait<Type = T> {} // ERROR
+trait Trait2<T> : Trait<Type = T> {} // ERROR: `???` requires `DefaultAutoTrait`, but `Self` is not `DefaultAutoTrait`
```
Comment (petrochenkov) [inline default auto traits.md:74]: It's not clear why this is an error.

Comment (Bryanskiy) [reply default auto traits.md:74]: ` DefaultAutoTrait` required for `Self` by bound in `Trait::Type`

Comment (Bryanskiy) [reply default auto traits.md:74]: I'm going to merge PR and fix comment with next commit. Ok?

Comment (petrochenkov) [reply default auto traits.md:74]: Ok.

**Pull request merged**